packages/core/src/tracing/w3c_trace_context/W3cTraceContext.js

```diff
@@ -51,6 +55,8 @@ W3cTraceContext.fromInstanaIds = function fromInstanaIds(instanaTraceId, instana
   traceContext.traceParentTraceId = paddedTraceId;
   traceContext.traceParentParentId = instanaParentId;
   traceContext.sampled = sampled;
+  // Instana tracers always generate trace IDs randomly, so we can confidently set this flag.
+  traceContext.randomTraceId = true;
```

Comment:
Suggestion: As this is a true or false state, I would like to suggest to call it `isRandomTraceId`.

Reply:
I'm -1 on this. `sampled` is also a boolean. The names for the flags are taken directly from the specification. I think this makes sense -- you probably need to read the spec anyway to know what this code is about and if you have read it, using the same name helps making the connection between the code and the concepts discussed in the spec easier.

Reply:
> The names for the flags are taken directly from the specification.

The name in the spec is "random" :) 

All good, was not important 👍

Reply:
> The name in the spec is "random" :)

Ha, right. Depends on which part of the spec you look at. There is a code example in "3.2.2.5 trace-flags" that refers to the flag simply as `random`. Section [3.2.2.5.2 Random Trace ID Flag](https://www.w3.org/TR/trace-context-2/#random-trace-id-flag) calls it "random-trace-id flag". That's what I was referring to. And yes, I _always_ need to have the last word in arguments, lolol.